src/test/java/bio/terra/controller/StudyValidationsTest.java

```diff
@@ -323,7 +322,7 @@ private void checkValidationErrorModel(String context,
         for (int i = 0; i < messageCodes.length; i++) {
             String code = messageCodes[i];
             assertThat(context + ": correct message code (" + i + ")",
-                details.get(i), startsWith(messageCodes[i]));
+                details.get(i), containsString(messageCodes[i]));
```

Comment:
Did you find a case that wasn't startsWith? My concern is that we will get hits where we get a mismatched code that contains the substring of another code.

Reply:
I changed the format of the exception string. I'll wrap it with something so it can prevent the substring match.

Reply:
Wrapped the code in quotes and updated the test along with a comment describing the change.